modules/installation/pages/hardware-requirements.adoc

```diff
@@ -11,54 +11,141 @@ For {smr} hardware requirements, see xref:retail:retail-requirements.adoc[].
-| Swap space:               | 3{nbsp}GB
+
+|
+| Swap space
+| 3{nbsp}GB
+
 |===
 
-[cols="1,1", options="header"]
+[cols="1,1,1", options="header"]
 .Server Hardware Requirements for IBM POWER8 or POWER9 Architecture
 |===
-| Hardware                 | Recommended
-| CPU                      | Minimum 4 dedicated cores
-| RAM:                     | _Test Server_ Minimum 8{nbsp}GB
-|                          | _Base Installation_ Minimum 16{nbsp}GB
-|                          | _Production Server_ Minimum 32{nbsp}GB
-| Disk Space:              | [path]``/`` Minimum 100{nbsp}GB
-|                          | [path]``/var/lib/pgsql`` Minimum 50{nbsp}GB
-|                          | [path]``/var/spacewalk`` Minimum 50{nbsp}GB per SUSE product and 360{nbsp}GB per Red Hat product
-|                          | [path]``/var/cache`` In accordance to products to be synchronized: 100{nbsp}MB per {suse} product, 1{nbsp}GB per Red Hat or other product.
+
+| Hardware
+| Details
+| Recommendation
+
+| CPU
+|
+| Minimum 4 dedicated cores
+
+| RAM
+| Test Server
+| Minimum 8{nbsp}GB
+
+|
+| Base Installation
+| Minimum 16{nbsp}GB
+
+|
+| Production Server
+| Minimum 32{nbsp}GB
+
+| Disk Space
+| [path]``/`` (root directory)
+| Minimum 100{nbsp}GB
+
+|
+| [path]``/var/lib/pgsql``
+| Minimum 50{nbsp}GB
+
+|
+| [path]``/var/spacewalk``
+| Minimum 50{nbsp}GB per SUSE product and 360{nbsp}GB per Red Hat product
+
+|
+| [path]``/var/cache``
+| Minimum 10{nbsp}GB.
+Add 100{nbsp}MB per {suse} product, 1{nbsp}GB per Red Hat or other product.
 Double the space if the server is an ISS Master.
-| Swap space:               | 3{nbsp}GB
+
+|
+| Swap space
+| 3{nbsp}GB
+
 |===
 
 
 
 == Proxy Hardware Requirements
 
-[cols="1,1", options="header"]
+[cols="1,1,1", options="header"]
 .Proxy Hardware Requirements
 |===
-| Hardware                 | Recommended
-| CPU                      | Minimum 2 dedicated 64-bit CPU cores
-| RAM:                     | _Test Server_ Minimum 2{nbsp}GB
-|                          | _Production Server_ Minimum 8{nbsp}GB
-| Disk Space:              | [path]``/`` _(root)_ Minimum 24{nbsp}GB
-|                          | [path]``/srv`` Minimum 100{nbsp}GB
-|                          | [path]``/var/cache`` _(Squid)_ Recommended: minimum 100{nbsp}GB
+
+| Hardware
+| Details
+| Recommendation
+
+| CPU
+|
+| Minimum 2 dedicated 64-bit CPU cores
+
+| RAM
+| Test Server
+| Minimum 2{nbsp}GB
+
+|
+| Production Server
+| Minimum 8{nbsp}GB
+
+| Disk Space
+| [path]``/`` (root directory)
+| Minimum 24{nbsp}GB
+
+|
+| [path]``/srv``
+| Minimum 100{nbsp}GB
+
+|
+| [path]``/var/cache`` (Squid)
+| Minimum 100{nbsp}GB
+
 |===
 
+
 {productname} Server stores packages in the [path]``/var/spacewalk/`` directory.
 Repository synchronization fails if this directory runs out of disk space.
 You can estimate how much space the [path]``/var/spacewalk/`` directory requires based on the number and type of clients and repositories you plan to mirror.
```

Comment:
It does not depend on the number of clients, IMO.  Maybe, it would be better to say:

> ... of _different_ clients ...